runtime/src/snapshot_utils.rs

```diff
@@ -1522,6 +1522,18 @@ pub fn bank_from_snapshot_archives(
     accounts_update_notifier: Option<AccountsUpdateNotifier>,
     exit: Arc<AtomicBool>,
 ) -> Result<(Bank, BankFromArchiveTimings)> {
+    info!(
+        "Loading bank from full snapshot: {}, and incremental snapshot: {:?}",
```

Comment:
Follow-up: do we want to clarify terminology here, i.e. "Loading bank from full snapshot **archive**:"